src/api/config/initializers/assets.rb

```diff
@@ -10,3 +10,7 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 Rails.application.config.assets.precompile += ['webui.js']
+
+# Don't compress css because we use sassc-rails compressor to compress
+# css which does not support tailwind functions
+Rails.application.config.assets.css_compressor = nil
```

Comment:
@rubhanazeem not sure if disabling css compressing is something we can do on production. I don't have any numbers, but this probably will increase load times on every view. Have you considered exchanging `sassc` with https://github.com/rails/dartsass-rails/ for example?